src/librustc_mir/dataflow/mod.rs

```diff
@@ -30,6 +30,7 @@ use self::move_paths::MoveData;
 
```

Comment:
btw, as drive-by improvement potential, it would be great if this module pointed to some good beginner->less-beginner resources for understanding dataflow.

Reply:
@ecstatic-morse Since @oli-obk r+ed, maybe this can be done as a follow up?

Reply:
Sounds good. I tentatively plan to do this in the `rustc` guide, then link to it from `dataflow/mod.rs`.

Reply:
There's currently a "to be written" entry in the [appendix](https://rust-lang.github.io/rustc-guide/appendix/background.html?highlight=dataflow#what-is-a-dataflow-analysis)

Reply:
Sounds great. 👍